### PR TITLE
LPD 27101 

### PR DESCRIPTION
**Pull request by kyle-miho**
Update of https://github.com/liferay-devtools/liferay-portal/pull/99
cc:  @ling-alan-huang @drewbrokke 

https://liferay.atlassian.net/browse/LPD-27101

Grabbed the contextName from the service.xml that is required to be under the closest src folder. This seemed to be the most consistent way to find a specific file that would contain the portletShortName

### CODE REVIEW

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:hotel

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaServiceImplCheck.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		serviceXmlPath = serviceXmlPath.resolve("service.xml");
+
+		File file = serviceXmlPath.toFile();
+
+		if (!file.exists()) {
+			return null;
+		}
+
+		return file;
+	}
```
Comment (drewbrokke) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaServiceImplCheck.java:114]: @kyle-miho there may be a utility in SourceFormatter that accomplishes this already.

@ling-alan-huang any leads?

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one comment to check @kyle-miho. Otherwise this looks good. Thanks!

**Comment by ling-alan-huang** [comment]
see https://github.com/liferay-devtools/liferay-portal/pull/120

**Pull request closed without merging**